src/client/mod.rs

```diff
@@ -7,7 +7,7 @@ use crate::{
     client::partition::PartitionClient,
     connection::{BrokerConnector, MetadataLookupMode, TlsConfig},
     protocol::primitives::Boolean,
-    topic::Topic,
+    topic::Topic, backoff::BackoffConfig,
```

Comment:
Newline, probably. Make sure to run `cargo fmt`

Reply:
Ran this and it stayed the same.